core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosClusterScheduler.scala

```diff
@@ -517,6 +517,9 @@ private[spark] class MesosClusterScheduler(
           .addAllResources(memResourcesToUse.asJava)
         offer.resources = finalResources.asJava
         submission.schedulerProperties.get("spark.mesos.executor.docker.image").foreach { image =>
+          val forcePullImage = submission.schedulerProperties
```

Comment:
Actually, I also introduced the functionality for CoarseMesosSchedulerBackend. Am I missing a spot?

Nevertheless, I extracted it into a method in MesosSchedulerBackendUtil.